packages/germanium/build.sh

```diff
@@ -0,0 +1,18 @@
+TERMUX_PKG_HOMEPAGE=https://github.com/matsuyoshi30/germanium
+TERMUX_PKG_DESCRIPTION="Generate image from source code"
+TERMUX_PKG_LICENSE="MIT"
+TERMUX_PKG_MAINTAINER="Raven Ravener <ravener.anime@gmail.com>"
+TERMUX_PKG_VERSION=0.3.0
+TERMUX_PKG_SRCURL=https://github.com/matsuyoshi30/germanium/archive/v$TERMUX_PKG_VERSION.tar.gz
+TERMUX_PKG_SHA256=44fc26bb263f409363618ab091b7ca877f11870edbff436efc5fe2d4e950beda
+ 
+termux_step_make() {
+	termux_setup_golang
+ 
+	cd $TERMUX_PKG_SRCDIR
```

Comment:
It should be possible to set `TERMUX_PKG_BUILD_IN_SRC=true` and not have to `cd $TERMUX_PKG_SRCDIR`